pyaedt/generic/clr_module.py

```diff
@@ -30,11 +30,11 @@
         json_file = os.path.abspath(os.path.join(pyaedt_path, "misc", "pyaedt.runtimeconfig.json"))
         load("coreclr", runtime_config=json_file, dotnet_root=os.environ["DOTNET_ROOT"])
         print("DotNet Core correctly loaded.")
-        if "Delcross" not in os.getenv("LD_LIBRARY_PATH", "") or "mono" not in os.getenv("LD_LIBRARY_PATH", ""):
+        if "mono" not in os.getenv("LD_LIBRARY_PATH", ""):
             warnings.warn("LD_LIBRARY_PATH needs to be setup to use pyaedt.")
             warnings.warn("export ANSYSEM_ROOT222=/path/to/AnsysEM/v222/Linux64")
```

Comment:
Should update this line for 232 as well, and maybe consider displaying the original message if on <232?

Reply:
at this point I don't know on which AEDT version the code is gonna run. so Either I put the message for everyone or I remove for everyone

Reply:
@isaacansys  I've modified the message considering the option of 23.2. If you have a better idea let me know.